python/pyspark/ml/tuning.py

```diff
@@ -829,13 +850,11 @@ def _to_java(self):
         :return: Java object equivalent to this instance.
         """
 
-        sc = SparkContext._active_spark_context
-        # TODO: persst validation metrics as well
         _java_obj = JavaParams._new_java_obj(
             "org.apache.spark.ml.tuning.TrainValidationSplitModel",
             self.uid,
             self.bestModel._to_java(),
-            _py2java(sc, []))
+            self.validationMetrics)
```

Comment:
This seems fine but out of curiosity why is the `_py2java` call no longer needed here?

Reply:
I think we will be converting _py2java here?
https://github.com/apache/spark/blob/9e42c52c7728250547aa044f8c28f93de9940460/python/pyspark/ml/wrapper.py#L60-L69

Reply:
I compared with _py2java here and without here, both cases the written metadata  file is same. I'll add _py2java here, for consistency.